Kitodo-DataFormat/src/test/java/org/kitodo/dataformat/access/MetsXmlElementAccessIT.java

```diff
@@ -46,12 +46,14 @@
 
 public class MetsXmlElementAccessIT {
 
+    private static final String TOP_XML = "top.xml";
+
     private static final File OUT_FILE = new File("src/test/resources/out.xml");
 
     private static final InputStreamProviderInterface INPUT_STREAM_PROVIDER = (uri, unused) -> {
         try {
-            return new FileInputStream(new File("src/test/resources/" + uri.getPath()));
-        } catch (FileNotFoundException e) {
+            return Files.newInputStream(Paths.get("src/test/resources/" + uri.getPath()));
+        } catch (IOException e) {
```

Comment:
What is going on here?

Reply:
The new Codacy quality assurance wanted this change. In short: `new FileInputStream` has a bug in the Java garbage collector, so you should not use it anymore. This change is necessary for the Codacy check to turn green.

Reply:
What about not opening one stream for all test? Instead of it, use for each test separated stream which is open and closed in the same method?

Reply:
No, I do not open a stream for all tests here. I pass _this function_ to the METS module so that _the METS module can open_ linked METS files independently, whenever it needs to. This is just a dummy here. In productive operation, I pass here a function that requests the lock and realizes the opening of the stream via the file management. But this has nothing to do with Codacy forcing me to stop using `FileInputStream` here.